pom.xml

```diff
@@ -151,11 +151,6 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
-			<dependency>
```

Comment:
I don't see why we have to live with the Boot managed version since it is a very old one, 2.5.1, from Sep 2016.  We should move to the latest